packages/storybook/src/nimble/chip/chip-matrix.stories.ts

```diff
@@ -0,0 +1,127 @@
+    ['Hide Icon', false]
+] as const;
+type ShowStartSlotIconState = (typeof showStartSlotIconStates)[number];
+const showStartSlotIconStatesOnlyIcon = showStartSlotIconStates[0];
+
+const labelStates = [
+    ['Short Label', 'Option Label'],
+    [
+        'Long Label',
+        'A very long label that will likely overflow the chip and require truncation with set width'
+    ]
+] as const;
+type LabelState = (typeof labelStates)[number];
+const labelStatesOnlyShort = labelStates[0];
+
+const widthStates = [
+    ['Default Width', ''],
+    ['Custom Width', 'width: 250px;']
+] as const;
+type WidthState = (typeof widthStates)[number];
+const widthStatesOnlyDefault = widthStates[0];
+
+const metadata: Meta = {
+    title: 'Tests/Chip',
+    parameters: {
+        ...sharedMatrixParameters()
+    }
+};
+export default metadata;
+
+// prettier-ignore
+const component = (
+    [disabledName, disabled]: DisabledState,
+    [appearanceName, appearance]: AppearanceState,
+    [removableName, removable]: RemovableStates,
+    [showStartSlotIconName, showStartSlotIcon]: ShowStartSlotIconState,
+    [labelName, label]: LabelState,
+    [widthName, width]: WidthState
+): ViewTemplate => html`
+    <div style="display: flex; flex-direction: column;">
+        <label style="color: var(${controlLabelFontColor.cssCustomProperty}); font: var(${controlLabelFont.cssCustomProperty})">
+            ${appearanceName}, ${removableName}, ${showStartSlotIconName}, ${labelName}, ${widthName}, ${disabledName ? `(${disabledName})` : ''} 
+        </label> 
+        <${chipTag}
+            appearance="${() => appearance}"
+            ?removable="${() => removable}"
+            ?disabled=${() => disabled}
+            style="margin-right: 8px; margin-bottom: 8px; ${() => width};">
+                ${when(() => showStartSlotIcon, html`<${iconKeyTag} slot="start"></${iconKeyTag}>`)}
+                ${label}
+        </${chipTag}>
+    </div>
+`;
+
+export const themeMatrix: StoryFn = createMatrixThemeStory(
+    createMatrix(component, [
+        disabledStates,
+        appearanceStates,
+        removableStates,
+        showStartSlotIconStates,
+        labelStates,
+        widthStates
+    ])
+);
+
+const interactionStates = cartesianProduct([
+    disabledStates,
+    appearanceStates,
+    removableStates,
+    [showStartSlotIconStatesOnlyIcon],
+    [labelStatesOnlyShort],
+    [widthStatesOnlyDefault]
+] as const);
+
+const interactionStatesHover = cartesianProduct([
+    disabledStates,
+    appearanceStates,
+    removableStates,
+    [showStartSlotIconStatesOnlyIcon],
+    [labelStatesOnlyShort],
+    [widthStatesOnlyDefault]
+] as const);
+
+export const interactionsThemeMatrix: StoryFn = createMatrixThemeStory(
+    createMatrixInteractionsFromStates(component, {
+        hover: interactionStatesHover,
+        hoverActive: [],
+        focus: interactionStates,
+        active: []
+    })
+);
+export const textCustomized: StoryFn = createMatrixThemeStory(
```

Comment:
The italic text looks clipped in the Text Customized story.

Reply:
Yeah, it is, and I talked with @jattasNI about this, and it's unclear to me what's causing it. The `button` doesn't exhibit this behavior, and even if I align the styles for the chip and the button (for the content part), I can't get rid of the cut-off. Interested in suggestions here.